resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -427,6 +426,7 @@ private[spark] class MesosClusterScheduler(
   }
 
   private def getDriverUris(desc: MesosDriverDescription): List[CommandInfo.URI] = {
+    val useFetchCache = desc.conf.get(config.ENABLE_FETCHER_CACHE)
```

Comment:
Can we make this in a backward-compatible way? For example, if the configuration is not given from the submission, Spark had better work in the old way.

Reply:
Ok, set it to turn fetcher cache on if specified in either dispatcher or submission